src/referencePackages/src/microsoft.win32.systemevents/4.7.0/microsoft.win32.systemevents.nuspec

```diff
@@ -22,8 +22,6 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
     <serviceable>true</serviceable>
     <dependencies>
       <group targetFramework=".NETCoreApp3.0">
-        <!-- Manually updated to resolve 3.1.0 prebuilt.
```

Comment:
I dropped this comment/customization because the code gen'd reference is not being changed.